CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+0.16.0 (2022-12-006)
+===
+
+Plan
+---
+
+* Refactor `MarkSweep` to work with both our native mark sweep policy and the malloc mark sweep policy backed by malloc libraries.
+
+Allocators
+---
+
+* Add `FreeListAllocator` which is implemented as a `MiMalloc` allocator.
+* Fix a bug in `ImmixAllocator` that alignment is properly taken into consideration when deciding whether to do overflow allocation.
+
+Policies
+---
+
+* Add `MarkSweepSpace`:
+  * It uses our native MiMalloc implementation in MMTk core.
+  * When the feature `malloc_mark_sweep` is enabled, it uses the selected mimalloc library to back up its allocation.
```

Comment:
It should be "selected malloc library" instead of "selected mimalloc library".

Reply:
Right. Thanks for pointing this out.